### DISCUSSION TITLE
earthaccess Hackdays: Starting February 6!

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#440] [Announcements]
The [NASA Openscapes Mentors](https://nasa-openscapes.github.io/mentors) are kicking off a series of bi-weekly Hackdays beginning next Tuesday, February 6th from 11:00 AM - 1:00 PM MST (6:00-8:00 PM UTC) focused on `earthaccess` development. We aim to foster new contributions through small group work aligning around specific topics or features.

Please check out our [Community Calendar](https://earthaccess.readthedocs.io/en/latest/calendar/) for meeting details, including Zoom room and time. 

Do you have suggestions on topics or areas of work for these upcoming Hackdays?

-  What Issues are you interested in contributing to? 
- What features should we be prioritizing? 
- For our library maintainers, would it be helpful for me to set up a Roadmap view of our project that we can utilize for new and existing feature work?

**Comment by JessicaS11** [comment]
@asteiker Please add me to the invite list for these. Also, was the time being moved to 12pm MST or not?

**Reply by asteiker** [reply]
@JessicaS11 Great! You should have just received an invite. We are sticking with 11-1 MST - I hope that works for you!

**Reply by JessicaS11** [reply]
It does!

**Comment by jhkennedy** [comment]
@asteiker me as well! I'll try and join for at least the second half, once I'm out of my 11:00-12:00 MST meeting.

**Reply by asteiker** [reply]
Great! You should have just received an invite. Glad you're able to join the second half!

**Comment by Rapsodia86** [comment]
Hi @asteiker! I am also interested!

**Reply by Rapsodia86** [reply]
@asteiker Oh no. I just checked my calendar and have another workshop on the 6th. Sorry! So, I would like to join the next one then!:)

**Reply by asteiker** [reply]
@Rapsodia86 No worries! I invited you to the 6th before I saw this, but I also invited you to the following hackday on Feb 20. Looking forward to meeting you!

**Reply by Rapsodia86** [reply]
Great! Got the invite and accepted for the 20th! 
Thanks:)

**Comment by betolink** [comment]
Can we include @jpswinski?

**Reply by asteiker** [reply]
I added him to the hackday series last week! Let me know if you don't see it in your calendar, @jpswinski.

**Comment by asteiker** [comment]
We look forward to our next hackday tomorrow! Please note that the time is shifted ahead one hour to accommodate the [Openscapes Community Call](https://openscapes.org/events/2024-02-20-github-calendaring/):

**Tuesday, February 20th from 12:00 PM - 2:00 PM MST (7:00-9:00 PM UTC)**

Last hackday we formed breakout groups around topics including end-user documentation, test environment support, and s3 access checks. Do you have other topics, Issues, or feature ideas you'd like to work on? Let us know!

**Comment by stefaniebutland** [comment]
Next earthaccess hackday today **Tuesday, March 5, 1 - 3 ET, 10- 12 PT**

Notes from [2024-02-20](https://github.com/nsidc/earthaccess/discussions/463)

**Comment by chuckwondo** [comment]
I'm interested in these hackdays. How do I join in? I'm interested in working on https://github.com/nsidc/earthaccess/issues/120 to begin with, but open to other things as well.

**Comment by stefaniebutland** [comment]
@chuckwondo come on over to zoom. Here for next 1.5 hrs.
Would you like a calendar invite for biweeklies? Always welcome; no obligation

**Reply by chuckwondo** [reply]
Thank you, yes, please!

**Reply by mfisher87** [reply]
Was great to have you join us today! :)

**Reply by chuckwondo** [reply]
Thank you. I thoroughly enjoyed it, and I look forward joining next time!

**Reply by mfisher87** [reply]
>come on over to zoom

@stefaniebutland @chuckwondo Looking from the outside, how did Chuck get the zoom info in the end? I don't see it in this discussion.

I'd like to update the top-level post to have a bolded call-to-action that explains exactly how to request to join and what to expect in response :)

**Reply by stefaniebutland** [reply]
> how did Chuck get the zoom info

I pasted the zoom link, then deleted it once Chuck joined since we don't want open zoom links for recurring use being public. 

> exactly how to request to join and what to expect in response

🤔 If people get added to the Google Cal invite, that has zoom link and link to the notes doc. We set that Cal invite so guests can add others and I think that has worked for @asteiker. Can you try to add @doug-newman-nasa that way? The idea is to enable any NASA Openscapes Mentor to add someone, rather than having only 2 gatekeepers.

**Reply by mfisher87** [reply]
> The idea is to enable any NASA Openscapes Mentor to add someone, rather than having only 2 gatekeepers.

I now know how to do this, I already had permissions and used the calendar.google.com interface to edit the meeting. I don't know much about how Google Calendar permissions work, but if I can do it other mentors should have permissions too, so just give it a shot and I think it will "just work"! :rocket: 

>I'd like to update the top-level post to have a bolded call-to-action that explains exactly how to request to join and what to expect in response :)

I updated the OP!

**Comment by mfisher87** [comment]
@stefaniebutland can you invite @doug-newman-nasa?

**Reply by doug-newman-nasa** [reply]
Yes, please!

**Reply by stefaniebutland** [reply]
invited you just now @doug-newman-nasa. Sorry for my delay!

**Reply by mfisher87** [reply]
Sorry folks, that was totally on me. Stef asked me to test in the thread above and I never even responded, despite intending to :laughing: 

Thanks for getting the important work done :bow:

**Comment by botanical** [comment]
Hello 👋 I am interested in joining the hackdays! Could I get an invite please? My email address is jennifer@developmentseed.org

**Reply by mfisher87** [reply]
You should have a calendar invite now. Looking forward to seeing you next week (or whenever you can make it :slightly_smiling_face:  )!

**Comment by ayushnag** [comment]
Hi all! Could I get an invite as well? Interested in making cloud optimization contributions to earthaccess. Email: ayushn1 [at] uw [dot] edu

**Reply by mfisher87** [reply]
:wave: I added you to the meeting. Can you let me know if you get an invite? Google Calendar took a weirdly long time to save the change.

**Reply by ayushnag** [reply]
Yes I got it, thank you!

**Comment by Mikejmnez** [comment]
Hello everybody! I recently learned about this am interested in joining on some of the hackdays! I am interested in cloud access to EarthData, with a particular scientific interest but not limited to the oceans. I have not yet browsed through all the issues and discussions, but looking forward to learn more about this project and the ways I can contribute. Email is mjimenez@opendap.org

**Reply by mfisher87** [reply]
Invite sent! I'm just on my way out to a dentist appointment, so I won't be able to help if that didn't work. Not sure who will be there, so blasting everyone I can think of in 1 second :) cc @asteiker @stefaniebutland  @betolink

**Reply by mfisher87** [reply]
Context I left out: we have a hack day starting in 20 minutes :)

**Reply by stefaniebutland** [reply]
Amy and I will be there! Join us at any upcoming earthaccess hackday, every 2 weeks! No obligation, but great way to get oriented among a kind group of people

**Reply by Mikejmnez** [reply]
Got it, thanks!  :) -- I won't be able to join likely until June, but I will be looking forward for these!

**Comment by Sherwin-14** [comment]
Hi all! I would like to join the hackday on 28th. Could I get an invite for this?  I'm interested in contributing documentation for this project ;)

**Reply by mfisher87** [reply]
Will be great to have you! I'll reach out on LinkedIn to get an email for a calendar invite.

**Comment by iam4tune** [comment]
Hello, I would like to be a part of this, can I get an invite please?

**Reply by mfisher87** [reply]
We'd love to have you! Can you share an email I can use to send the calendar invite? If you'd like to share the email with me privately we can [connect on LinkedIn](https://www.linkedin.com/in/mattfisher8/), or you can post and then delete your email from this thread once you receive an invitation?

**Reply by mfisher87** [reply]
Invitation sent! If you received it OK, feel free to edit your email out or delete that comment :)

**Reply by iam4tune** [reply]
seen

**Comment by wykhuh** [comment]
I'm interested in joining. Can I get an invite?

**Reply by mfisher87** [reply]
Sent you a note on LinkedIn to request your email, if you don't get an invite by tomorrow please @ me here and I'll follow up!

**Reply by mfisher87** [reply]
Invitation sent :heavy_check_mark: Looking forward to seeing you!

**Comment by DeanHenze** [comment]
Hi all! Dean here from PODAAC, I'd like to help pick up where Ayush left off with cloud optimized access. Could I get an invite as well? @mfisher87 I messaged you on LinkedIn.

**Reply by mfisher87** [reply]
Amazing, sent you an invitation! Next meeting is August 6th at 11AM Mountain Time. Let me know if you have any trouble with the invitation. Looking forward to seeing you soon, Dean!

**Comment by nbharwani-cu** [comment]
Hello, I'd love to join and be apart of Hackaday @mfisher87 ! I'd love an invite for the biweekly meeting in case I can't make the next one. I haven't looked through the open issues, but would like to find ways I can contribute. I'm looking for an issue that might help get me familiar with the process and something more manageable as a junior developer. Paired programming is always welcomed! I connected via LinkedIn with my email.

**Reply by mfisher87** [reply]
@NaureenBharwaniNOAA Awesome, I'll add you to the calendar invite!

The next meeting is today in only 45 minutes!

**Comment by rwegener2** [comment]
Hi earthaccess team! I'm interested in joining the hacktime. Can I please get an invite? @mfisher87 I just messaged you on the Openscapes slack with my email.

**Reply by mfisher87** [reply]
Invite sent! Thanks for your interest :hugs:

**Comment by kylebarron** [comment]
👋 Would you be able to add me to the invite? kyle@developmentseed.org

**Reply by mfisher87** [reply]
:wave: Hi Kyle! Invite sent!

**Comment by anamcilie** [comment]
I am also interested! anamcilie@gmail.com

**Reply by mfisher87** [reply]
Amazing, Ana! Invite sent!

**Comment by ana-sher** [comment]
Hello, I am interested as well. Could I get an invite, please? nshcherbakovaa@gmail.com

**Reply by asteiker** [reply]
Hi Anastasiia, I just sent the invite. We look forward to seeing you at our next hackday!

**Comment by kyradegen** [comment]
Hello, I am interested a well. Could I get an invite? kdegen@contractor.usgs.gov

**Reply by jules32** [reply]
Hi Kyla! Sorry we missed this and thanks Erik Bolch for boosting. I've just added you to the calendar invites, welcome! Excited for you to join and that's so awesome you're interested in working on the cheatsheet, zine, and documentation 🌟
